modules/gltf/test/lib/gltf-scenegraph-modifiers.spec.js

```diff
@@ -3,6 +3,8 @@ import test from 'tape-promise/tape';
 
 import {GLTFScenegraph} from '@loaders.gl/gltf';
 
+import {loadI3STileContent} from '../../../i3s/test/lib/utils/load-utils';
```

Comment:
We don't do imports like this across modules. There are aliases, you can import `@loaders.gl/i3s/test/...`.